src/components/ItaliaTheme/View/Commons/PageHeader/PageHeader.jsx

```diff
@@ -39,104 +37,112 @@ const messages = defineMessages({
 });
 
 const PageHeader = (props) => {
```

Comment:
```suggestion
/**
 * @params content {object} content object
 * @params imageinheader {boolean} if true, show image in header (eg. in PersonaView)
 * @params imageinheader_field {string} field name of the image to show in header (eg. 'foto_persona')
 */
const PageHeader = (props) => {
```
Mi sembra che la sintassi sia questa, poi scriverla qui nell'editor dei commenti non è perfetto quindi non vedo l'output ma l'editor ti aiuta a sistemare in caso